apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep4.vue

```diff
@@ -135,13 +142,23 @@ const handleClose = () => {
     state.deleteModalVisible = false;
 };
 
-const handleConfirmCreateCollector = () => {
-    // TODO: logic for collect now
-    state.visibleCreateModal = false;
+const handleConfirmCreateCollector = async () => {
+    try {
+        state.collectLoading = true;
+        await SpaceConnector.client.inventory.collector.collect({
+            collector_id: state.createdCollectorId,
+        });
```

Comment:
nitpick: Oh, I thought it was a mistake. What about give a comment like "after the collector created, first collection must be executed" ?